src/problem.cpp

```diff
@@ -165,21 +204,21 @@ std::vector<Solution> Problem::FindSolutionsImpl(Handle& handle,
                                                  const AllocatedBuffers& buffers,
                                                  const ConvolutionDescriptor& conv_desc) const
 {
-    auto ret = std::vector<Solution>{};
+    const auto& actual = conv_desc.mode == miopenTranspose ? Transpose() : *this;
+    auto ret           = std::vector<Solution>{};
 
     if(tensor_descriptors.size() != 3)
         MIOPEN_THROW(miopenStatusInvalidValue,
                      "Convolution problem should have exactly three tensor descriptors.");
 
+    // These are not swapped for now to preserve argument order in calls
```

Comment:
Wow. It seems like we do not have a proper design for transposed convolutions. Let's discuss.

Reply:
This would get cleaned up in the next find 2.0 PR using the new `FindConvolution` method from #1739 rather than `Find...Convolution`, which are other API mappings and should only be used as such.
These are not swapped to make calls structure exactly same as in other API mappings until the whole approach would be changed very soon.